development/config.rb

```diff
@@ -2,7 +2,7 @@
 AppConfig[:emory_ga_tracking_public] = ''
 AppConfig[:pui_branding_img] = '/assets/images/shield.svg'
 AppConfig[:pui_branding_img_alt_text] = 'Emory University Libraries Logo'
-AppConfig[:emory_plugins_version] = 'v0.2.1'
+AppConfig[:emory_plugins_version] = 'v0.2.2'
```

Comment:
This version is not the version of the plugin, but rather the version of this deployment repository that we need to create. So we need to update it to `v2.2.3` for all config files. Then create a new release in this repository with tag `v2.2.3`. Also since we are updating only dev and test, only dev and test config files should be changed.